packages/cli/src/commands/docgen.ts

```diff
@@ -35,9 +35,11 @@ const commandToPathMap: Record<string, string> = {
 
 export type DocType = keyof typeof commandToPathMap;
 
+// A list of UNIQUE possible default filenames for the polywrap manifest
 const defaultManifest = defaultPolywrapManifest
   .concat(defaultAppManifest)
-  .concat(defaultPluginManifest);
+  .concat(defaultPluginManifest)
+  .filter((value, index, self) => self.indexOf(value) === index);
```

Comment:
This has been added because now the `app` and `plugin` manifest can be `polywrap.yaml` - Correct?

Reply:
Correct!

Reply:
This is a clever way to get a unique list!